extensions/sql-database-projects/src/common/azureFunctionsUtils.ts

```diff
@@ -0,0 +1,73 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import * as utils from './utils';
+import * as constants from './constants';
+import { parseJson } from './parseJson';
+
+/**
+ * Represents the settings in an Azure function project's local.settings.json file
+ */
+export interface ILocalSettingsJson {
+	IsEncrypted?: boolean;
+	Values?: { [key: string]: string };
+	Host?: { [key: string]: string };
+	ConnectionStrings?: { [key: string]: string };
+}
+
+/**
+ * copied and modified from vscode-azurefunctions extension
+ * @param localSettingsPath full path to local.settings.json
+ * @returns settings in local.settings.json
+ */
+export async function getLocalSettingsJson(localSettingsPath: string): Promise<ILocalSettingsJson> {
+	if (await fse.pathExists(localSettingsPath)) {
```

Comment:
What about prompting the user to select the file if we can't find it in the default location?

Reply:
I was thinking if the user already has to go look for the location of their local.settings.json, they might as well just copy the setting name from there or they can just put a placeholder value for now.